lib/main.js

```diff
@@ -2,6 +2,7 @@ var assign = require('object-assign')
 var path = require('path')
 var BrowserWindow = require('browser-window')
 var wargs = require('./args')
+var v = (process.versions.electron || '').split('.').map(Number)
```

Comment:
Yes, absolutely! But here the code is supposed to be backwards compatible and the tests are currently run with mocha in node (so we'd have to enable `--harmony_destructuring` or use babel).

Besides, I would hope that these version checks can be removed soon anyway, seeing how fast Electron is moving along.